src/App.tsx

```diff
@@ -1,61 +1,96 @@
+import { useState } from 'react';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUser = (userId: number): User | null => usersFromServer
+  .find(user => user.id === userId) ?? null;
 
 export const App = () => {
+  const [todoList, setTodoList] = useState(
+    todosFromServer.map(todo => ({ ...todo, user: getUser(todo.userId) })),
```

Comment:
Try to not pass calculations to useState. It's better to calculate this value before